pkg/machines/libvirt-dbus.es6

```diff
@@ -315,6 +317,47 @@ LIBVIRT_DBUS_PROVIDER = {
         };
     },
 
+    DETACH_DISK({
+        name,
+        connectionName,
+        id: vmPath,
+        target,
+        live
+    }) {
+        /*
+         * 0 -> VIR_DOMAIN_AFFECT_CURRENT
+         * 1 -> VIR_DOMAIN_AFFECT_LIVE
+         * 2 -> VIR_DOMAIN_AFFECT_CONFIG
```

Comment:
What does "CURRENT" mean? It sounds like the same as LIVE, and _CONFIG is the on-disk one that gets applied to the next VM run only.

Reply:
Copying from libvirt docs.

VIR_DOMAIN_AFFECT_CURRENT | = | 0 | Affect current domain state.
VIR_DOMAIN_AFFECT_LIVE | = | 1 | Affect running domain state.
VIR_DOMAIN_AFFECT_CONFIG | = | 2 | Affect persistent domain state

So VIR_DOMAIN_AFFECT_CURRENT is like use VIR_DOMAIN_AFFECT_LIVE if domain is running and as VIR_DOMAIN_AFFECT_CONFIG if domain is shutoff.